samples/ControlCatalog/Pages/DialogsPage.xaml.cs

```diff
@@ -397,7 +397,7 @@ private static string FullPathOrName(IStorageItem? item)
         }
 
         Window GetWindow() => this.VisualRoot as Window ?? throw new NullReferenceException("Invalid Owner");
```

Comment:
```suggestion
        Window GetWindow() => TopLevel.GetTopLevel(this) as Window ?? throw new NullReferenceException("Invalid Owner");
```
just for a consistency